src/main/java/seedu/duke/FoodPlace.java

```diff
@@ -0,0 +1,18 @@
+package seedu.duke;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+public class FoodPlace {
+
+    private ArrayList<String> faculty = new ArrayList<>();
+    private ArrayList<DiningOptions> info = new ArrayList<>();
+
+    public FoodPlace(String[] faculty, String[][] data) {
+        this.faculty.addAll(Arrays.asList(faculty));
+        for (int i = 0; i < data.length; i++) {
+            info.add(new DiningOptions(data[i]));
+        }
+    }
+
```

Comment:
Perhaps we should add getters for faculty and info here too

Reply:
Sure! I was wondering if those should be added if and when they are being used...

Reply:
Sure, we can add it when implementing the other features